addons/captives/functions/fnc_doBlindfoldCaptive.sqf

```diff
@@ -0,0 +1,73 @@
+#include "script_component.hpp"
+/*
+ * Author: mrschick, johnb43
+ * Puts a blindfold on a captive unit if the player has a blindfold in their inventory.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Target <OBJECT>
+ * 2: Put on (true) or take off (false) <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, cursorTarget, true] call ace_captives_fnc_doBlindfoldCaptive
+ *
+ * Public: No
+ */
+
+params ["_unit", "_target", "_state"];
+
+if (_state) then { // Blindfold target
+    // Check if _unit has a blindfold in its inventory, abort otherwise.
+    private _carriedBlindfoldIdx = GVAR(blindfolds) findAny (_unit call EFUNC(common,uniqueItems));
+    if (_carriedBlindfoldIdx == -1) exitWith { ERROR("no blindfold"); };
```

Comment:
I'm not sure if `ERROR` is needed/wanted.
Thoughts?

Reply:
[PabstMirror added it](https://github.com/acemod/ACE3/commit/e724b3e06363596d137437694cbeab8f7d334e31). I agree that it belongs in here because during normal execution, if `_unit` does not have a blindfold then `doBlindfoldCaptive` should not have been called in the first place.